### PR TITLE
Describe what code does and minor fix

### PR DESCRIPTION
**Pull request by mikikiv**
Added description to what the example code does: change from yellow to orange over 5 seconds
fix: Added that the animation duration must be defined for the animation to work.

Explaining what the example code does is helpful for learning!

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by Dusch4593** [comment]
Hey @mikikiv! Hope you're doing well!

Me and a few of my colleagues are hosting [a live demo of Codecademy Docs on a livestream event this Friday](https://www.codecademy.com/events/demo-learn-about-codecademy-docs-part-2) where we'll showcase the site/codebase and review a few PRs to demonstrate our review process and encourage more/new contributions.

May we use your recent contribution, this PR, as one of the PRs to review this Friday? We're hoping to have it reviewed and merged that day.

**Comment by mikikiv** [comment]
@Dusch4593 Yes, absolutely! I watched the livestream last week and it's what helped me find this place and look for something to contribute on. Thanks for having this place for the community to teach and learn

**Comment by Dusch4593** [comment]
Thank YOU @mikikiv!

File: content/css/concepts/animations/animations.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ To use CSS animation, some keyframes for the animation must be specified.
 
 Keyframes hold what CSS styles the element will have at certain times. When you specify styles inside the `@keyframes` rule, the animation will gradually change from the current styles to the new styles.
 
-To get an animation to work, the animation must be bound to an element:
+To get an animation to work, the animation must be bound to an element and the animation-duration must be defined. In this example code the `<h1>` elemenent will load as yellow and gradually change to orange over the course of 5 seconds.
```
Comment (sonnynomnom) [inline content/css/concepts/animations/animations.md:23]:
```suggestion
To get an animation to work, the animation must be bound to an element and the `animation-duration` must be defined. In this example code, the `<h1>` element will load as yellow and gradually change to orange over the course of 5 seconds.
```

**Review by sonnynomnom — COMMENTED** [review]
One little change @mikikiv!

**Review by yangc95 — APPROVED** [review]
I made a [change](https://github.com/Codecademy/docs/pull/455/commits/9a18146e202666be82dcae39a20885aa1853806d) to the description.

In any case, LGTM for merging 👍

**Comment by sonnynomnom** [comment]
Merged Mikey! It's live here: https://www.codecademy.com/resources/docs/css/animations

Also, make sure to update your Codecademy profile pic :)

**Comment by Dusch4593** [comment]
> Also, make sure to update your Codecademy profile pic :)

> A few things... First, make sure to add a profile picture to your Codecademy profile :)

Hi, @mikikiv!

Fwiw, here are some steps to do just that.

1. Log in to your Codecademy account.
2. Near the top-right of the page, click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. Then click "Choose Photo".
5. Attach/save photo.
6. Click "Save Changes".

**Pull request merged**